DuggaSys/microservices/shared_microservices/getUid_ms.php

```diff
@@ -0,0 +1,28 @@
+<?php
+    date_default_timezone_set("Europe/Stockholm");
+    
+    // Include basic application services
+    include_once ("../../../../coursesyspw.php");
+    include_once ("../../../Shared/sessions.php");
+    include_once ("../../../Shared/basic.php");
+
+    function getUid(){
+        // Checks user id, if user has none a guest id is set
+        if(isset($_SESSION['uid'])){
+            $userid=$_SESSION['uid'];
+        }else{
+            $userid="1";
+        }
+
+        $log_uuid = getOP('log_uuid');
+        $log_timestamp = getOP('log_timestamp');
+
+        $log_uuid = getOP('log_uuid');
+        $info="opt: ".$opt." courseId: ".$courseId." courseVersion: ".$courseVersion." exampleName: ".$exampleName." sectionName: ".$sectionName." exampleId: ".$exampleId;
```

Comment:
Is this tested? The variables inside the string aren't defined/declared earlier and shouldn't have data available resulting in a empty string (or maybe a crash, since the variables aren't defined? not sure how PHP handles this).

Reply:
I have confirmed that this doesn't work. When echoing out the `$info` values, they are empty. 
![image](https://github.com/HGustavs/LenaSYS/assets/129070844/b0608864-d4a7-4e8a-8e46-5073a5314fd9)

If these are needed for the logging, then they should accessed before `$info` variable is declared/defined.